test/e2e/tests/dapp-interactions/block-explorer.spec.ts

```diff
@@ -89,7 +93,7 @@ describe('Block Explorer', function () {
 
         // Verify block explorer
         await driver.waitForUrl({
-          url: `https://etherscan.io/token/0x581c3C1A2A4EBDE2A0Df29B5cf4c116E42945947`,
+          url: `https://etherscan.io/token/${contractAddress.toLowerCase()}`,
```

Comment:
we need to lower case the acontract address as in anvil we get the checksummed value instead